guides/installing_esphome.rst

```diff
@@ -0,0 +1,97 @@
+Installing ESPHome
+==================
+
+Windows
+-------
+
+Download Python from `the official site <https://www.python.org/downloads/>`_.
+
+.. figure:: images/python-win-installer.png
+    :align: center
+    :width: 75.0%
+    :alt: Python installer window with arrows pointing to "Add Python to PATH" and "Install Now"
+
+Make sure you check "Add Python to PATH", and go all the way through the
+installer.
+
+Log out and back in, or restart your computer. Whichever is easiest.
+
+Open the start menu and type ``cmd``. Press the enter key.
+
+In the terminal that comes up, check that Python is installed:
+
+.. code-block:: console
+
+    > python --version
+    Python 3.10.1
+
+.. note::
+
+    Don't copy the ``>``. That's used to show that this is a command that goes
+    in the console, and to let you see what the expected results are (shown on
+    the next line without a ``>``)
+
+Looks good? You can go ahead and install ESPHome:
+
+.. code-block:: console
+
+    > pip3 install wheel
+    > pip3 install esphome
+
+And you should be good to go! You can test that things are properly installed
+with the following:
+
+.. code-block:: console
+
+    > esphome version
+    Version: 2021.12.3
+
+Mac
+---
```

Comment:
Having an empty section is weird. We should either add a note that there's no instructions yet and we welcome contributions with them to the docs, or leave it out completely.

Reply:
I've added a note that contributions are welcome, as well as pointed the user in the general right direction to get it working.